src/WLGDPSLocation.cc

```diff
@@ -5,14 +5,12 @@
 ////////////////////////////////////////////////////////////////////////////////
 // Description:
 //   This is a primitive scorer class for scoring energy deposit location
-//   at termination of the track.
 //
 ///////////////////////////////////////////////////////////////////////////////
 
 WLGDPSLocation::WLGDPSLocation(G4String name, G4int depth)
 : G4VPrimitiveScorer(name, depth)
```

Comment:
```suggestion
: G4VPrimitiveScorer(std::move(name), depth)
```

avoids an extra copy